README.md

```diff
@@ -1,12 +1,12 @@
-# Waste permits acceptance tests
+# Waste carriers acceptance tests
```

Comment:
You need to update the config section of the README as well; it contains a lot of stuff related to Waste permits and not waste carriers.